include/boost/geometry/strategies/geographic/area_geographic.hpp

```diff
@@ -0,0 +1,212 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2016 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_AREA_GEOGRAPHIC_HPP
+#define BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_AREA_GEOGRAPHIC_HPP
+
+#include <boost/geometry/formulas/area_formulas.hpp>
+#include <boost/geometry/formulas/thomas_inverse.hpp>
+//#include <boost/math/special_functions/atanh.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace area
+{
+
+/*!
+\brief Geographic area calculation by trapezoidal rule plus integral
+approximation that gives the ellipsoidal correction
+
+}
+
+*/
+
+template
+<
+    typename PointOfSegment,
+    typename Strategy = void,
+    const std::size_t SeriesOrder = 2,
+    bool ExpandEpsN = true,
+    bool LongSegment = false,
+    typename Spheroid = void,
+    typename CalculationType = void
+>
```

Comment:
I changed the interface so now the strategy can be defined as `area_geographic <PointType, InverseFormula, Order>` where InverseFormula can be one of `bg::formula::vincenty_inverse, bg::formula::thomas_inverse, bg::formula::andoyer_inverse` and Order a number in [0,5].